ci/buildspec.yml

```diff
@@ -9,10 +9,11 @@ phases:
     commands:
     - echo Build started on `date`
     - echo Building the Docker image...
-    - docker build -t sagemaker-sparkml-serving:2.4 .
-    - docker tag sagemaker-sparkml-serving:2.4 515193369038.dkr.ecr.us-west-2.amazonaws.com/sagemaker-sparkml-serving:2.4
+    - ls
```

Comment:
curious why this is in here, attempting to get the directory listing in the pipeline ligs?

Reply:
That was placed for testing purpose. I will be removing it.